support/cas-server-support-aup-core/src/main/java/org/apereo/cas/aup/BaseAcceptableUsagePolicyRepository.java

```diff
@@ -133,11 +133,18 @@ protected boolean isUsagePolicyAcceptedBy(final Principal principal) {
      */
     protected boolean isUsagePolicyAcceptedBy(final Map<String, List<Object>> attributes) {
         val core = aupProperties.getCore();
+
         if (attributes != null && attributes.containsKey(core.getAupAttributeName())) {
             val value = CollectionUtils.toCollection(attributes.get(core.getAupAttributeName()));
             LOGGER.debug("Evaluating attribute value [{}] found for [{}]", value, core.getAupAttributeName());
             return value.stream().anyMatch(v -> v.toString().equalsIgnoreCase(getAcceptedAttributeValue()));
         }
+
+        if (core.isAupOmitIfAttributeMissing()) {
+            LOGGER.debug("Omitted with attributes: [{}]", attributes);
```

Comment:
Switch this to TRACE, also please do no log the attributes, if possible. Instead, explain the behavior in the log message more clearly.

Reply:
Agree. Changed.